src/collection-browser.ts

```diff
@@ -869,7 +903,8 @@ export class CollectionBrowser
   handleResize(entry: ResizeObserverEntry): void {
     const previousView = this.mobileView;
     if (entry.target === this.contentContainer) {
-      this.mobileView = entry.contentRect.width < this.mobileBreakpoint;
+      this.contentWidth = entry.contentRect.width;
+      this.mobileView = this.contentWidth < this.mobileBreakpoint;
```

Comment:
Preserving the contentRect's width to be passed down into the date picker (which requires an explicit width to render its svg).